src/convertToObject.js

```diff
@@ -3,16 +3,36 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  let properties = sourceString.split(';');
```

Comment:
Split sourceString by '; ' instead of ';' to avoid extra whitespaces.

Reply:
It doesn't work. There isn't space between the first and second lines; Maybe I'm not right. 
`background-color:      #fff;
-webkit-border-radius: 5px;` 

Here is an example.
`["
  background-color:      #fff", "
-webkit-border-radius: 5px", "
  border-radius     : 5px", "
  border: 1px solid #e8e8e8", "
  
"]
["
  background-color:      #fff;
-webkit-border-radius: 5px;
  border-radius     : 5px;
  border: 1px solid #e8e8e8;
  
"]`